src/views/AmalgamationRegular/ReviewConfirm.vue

```diff
@@ -179,63 +179,64 @@
       </v-card>
     </section>
 
-    <!-- Certify -->
+    <!-- Court Order and Plan of Arrangement -->
     <section
-      id="certify-section"
+      v-if="isRoleStaff"
```

Comment:
I just noticed that this section isn't staff-only for IAs. Please confirm with Yui.

Reply:
Yui confirmed Staff only for Amalgamations.

Reply:
Should it be staff-only for incorporations?

Reply:
No. From Yui: It's weird for this one because the Statement of Amalgamation (visible non-staff) mentions court order but Court Order component should only be visible to staff. The 2 components were reviewed and revised with staff, co-design like. We may need them to take a look again when all the interactions are done. It feels ... odd. But let's just impl what we got